test/cart_test_ep_credits.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+# Copyright (C) 2018-2019 Intel Corporation
```

Comment:
only 2019. same comment for other new files.

Reply:
These files were copied from existing tests, so have to keep 2018 copyright on them